Explicitly state integer width

Rio6 opened the pull request:
This fixes timestamp and altitude message, which were pushing incorrect data before.

Also added a convery script to convert parsely output into csv.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5)
<!-- Reviewable:end -->

Rio6 committed:
explicit specify integer width and fix altitude and timestamp data

Rio6 committed:
add a convert script to parse parsely output

BluCodeGH reviewed (changes requested):
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5)* status: all files reviewed, 4 unresolved discussions (waiting on @QuantumManiac, @Rio6, @wendi-yu, @ZTeper, and @zx100x100)

___
*[gps_module.c, line 30 at r1](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5#-MwjPpeH5s1TRKqVE0rt:-MwjPpeH5s1TRKqVE0ru:b-taog2o) ([raw file](https://github.com/waterloo-rocketry/cansw_gps/blob/48569c8a704a6ea0a9f142d31ad0820c0c43110b/gps_module.c#L30)):*
> ```C
> 
> // message contents, stored locally
> //I've commented out the placeholders that I've been using
> ```

Out of date comment?
___
*[gps_module.c, line 79 at r1](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5#-MwjOycADpm6iexS4ITD:-MwjOycADpm6iexS4ITE:bawuzsb) ([raw file](https://github.com/waterloo-rocketry/cansw_gps/blob/48569c8a704a6ea0a9f142d31ad0820c0c43110b/gps_module.c#L79)):*
> ```C
>     strtodec(parser.msg, 10, &utc, &dsec);
> 
>     build_gps_time_msg(millis(), (uint8_t) (utc / 10000 % 100), (uint8_t) (utc / 100 % 100), (uint8_t) (utc % 100), (uint8_t) (dsec / 100), &msg_utc);
> ```

Why are we dividing dsec by 100 here? Is the intent to only track to the nearest 10th of a second? Overall the functions in the file could use comments explaining what their goal is.
___
*[utils/convert.py, line 48 at r1](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5#-MwjQUL3EkOcoNnFCzFA:-MwjQUL3EkOcoNnFCzFB:b7orp7g) ([raw file](https://github.com/waterloo-rocketry/cansw_gps/blob/48569c8a704a6ea0a9f142d31ad0820c0c43110b/utils/convert.py#L48)):*
> ```Python
>         return (
>             self.timestamp is not None and
>             #self.alt is not None and
> ```

Why is this commented out?
___
*[utils/convert.py, line 54 at r1](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5#-MwjQZ2T2YIfDcGbfkuD:-MwjQZ2T2YIfDcGbfkuE:bvlyxln) ([raw file](https://github.com/waterloo-rocketry/cansw_gps/blob/48569c8a704a6ea0a9f142d31ad0820c0c43110b/utils/convert.py#L54)):*
> ```Python
> 
>     def print_header(self):
>         #print(f'timestamp, alt, lat, lon')
> ```

Likewise, best to at least explain why altitude was removed.


<!-- Sent from Reviewable.io -->

Rio6 committed:
update comments

Rio6 commented:
I commented altitude part out from the converter script because gpsvisualizer doesn't use it, and I don't want to add another variable while debugging. I added it back now debugging is done.

Also updated the comments.

BluCodeGH reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/cansw_gps/5#-:-Mwrnnd8DqheUfKiDj_Y:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @QuantumManiac, @wendi-yu, @ZTeper, and @zx100x100)



<!-- Sent from Reviewable.io -->
